specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -5015,6 +5015,21 @@
         }
       }
     },
+    "GitHubClientSecret": {
+      "x-ms-discriminator-value": "GitHubClientSecret",
```

Comment:
FYI: If this matches the class name, this is not necessary.

Reply:
removed unnecessary `x-ms-discriminator-value`